boot/src/test/java/com/taobao/arthas/boot/DownloadUtilsTest.java

```diff
@@ -38,4 +51,97 @@ public void testReadMavenMetaData() throws IOException {
         Assert.assertNull(readMavenMetaData("", false));
         Assert.assertNull(readMavenMetaData("https/", false));
     }
+
+	// Test written by Diffblue Cover.
+
+	@Test
+	public void downArthasPackagingInputNullFalseNullNullOutputNullPointerException()
+			throws IOException, SAXException, ParserConfigurationException {
+
+		// Act
+		thrown.expect(NullPointerException.class);
+		DownloadUtils.downArthasPackaging(null, false, null, null);
+
+		// The method is not expected to return due to exception thrown
+	}
+
+	// Test written by Diffblue Cover.
+
+	@Test
+	public void getRepoUrlInputNotNullFalseOutputNotNull() {
+
+		// Arrange
+		final String repoMirror = "a,b,c";
+		final boolean http = false;
+
+		// Act
+		final String actual = DownloadUtils.getRepoUrl(repoMirror, http);
+
+		// Assert result
+		Assert.assertEquals("a,b,c", actual);
+	}
+
+	// Test written by Diffblue Cover.
+	@Test
+	public void getRepoUrlInputNotNullFalseOutputNotNull2() {
+
+		// Act and Assert result
+		Assert.assertEquals(
+				"https://repo1.maven.org/maven2",
+				DownloadUtils.getRepoUrl(
+						"                         center\u0001\u0001   \u0000\u0001 \u0001\u0001    \u0003\u0001  \u0003 \u0003\u0002\u0001\u0001 \u0003 \u0001 \u0002  \u0003\u0001  \u0001\u0001\u0001\u0000\u0000\u0000\u0000\u0000",
```

Comment:
This is worth reporting as a bug for when the new feature of inlining goes wrong.

Reply:
Once a bug has been reported it will be OK to refactor this.